src/components/ui/dialog.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from "react"
-import * as DialogPrimitive from "@radix-ui/react-dialog"
+import DialogPrimitive from "@radix-ui/react-dialog"
```

Comment:
this is right for loading natively but wrong for vite and vice versa; what do we do here?

Reply:
@geoffreylitt investigate